modules/ducktests/tests/ignitetest/services/utils/ignite_spec.py

```diff
@@ -102,12 +102,13 @@ class IgniteNodeSpec(IgniteSpec, IgnitePersistenceAware):
     """
     @property
     def command(self):
-        cmd = "%s %s %s %s 2>&1 | tee -a %s &" % \
+        cmd = "%s %s %s %s 2>&1 | tee %s >> %s &" % \
               (self._envs(),
                self.path.script("ignite.sh"),
                self._jvm_opts(),
                self.CONFIG_FILE,
-               self.STDOUT_STDERR_CAPTURE)
+               self.STDOUT_STDERR_CAPTURE,
+               self.CONSOLE_ALL_CAPTURE)
```

Comment:
why not just add some separator on the restart and continue to use the same log?
more logs to the god of logs? %)

Reply:
in the chat discussion, we decided to do so